src/main/java/org/spongepowered/api/data/property/Properties.java

```diff
@@ -0,0 +1,384 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.data.property;
+
+import com.flowpowered.math.vector.Vector3d;
+import org.spongepowered.api.block.BlockState;
+import org.spongepowered.api.block.BlockType;
+import org.spongepowered.api.block.BlockTypes;
+import org.spongepowered.api.data.manipulator.mutable.entity.DominantHandData;
+import org.spongepowered.api.data.type.ArmorType;
+import org.spongepowered.api.data.type.HandPreference;
+import org.spongepowered.api.data.type.InstrumentType;
+import org.spongepowered.api.data.type.Matter;
+import org.spongepowered.api.data.type.ToolType;
+import org.spongepowered.api.effect.potion.PotionEffect;
+import org.spongepowered.api.effect.sound.record.RecordType;
+import org.spongepowered.api.entity.living.Living;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.item.ItemType;
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.api.item.inventory.equipment.EquipmentType;
+import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+
+import java.util.Set;
+
+@SuppressWarnings("unchecked")
+public final class Properties {
+
+    public static final class Block {
+
+        // SORTFIELDS:ON
+
+        /**
+         * A property to represent the blast resistance of a block.
+         */
+        public static final DoubleProperty BLAST_RESISTANCE =
+                DummyObjectProvider.createFor(DoubleProperty.class, "BLAST_RESISTANCE");
```

Comment:
Extra line difference?

Reply:
The lines are sometimes too long, so I moved it to a new line for everything.